lib/documents.js

```diff
@@ -318,8 +314,9 @@ async function _hmacDocument({hmac, document}) {
 
 async function _hmacString({hmac, value}) {
   const data = TEXT_ENCODER.encode(value);
-  const signature = await hmac.sign({data});
-  // TODO: worth using hmac type to add prefix to signature for future proofing?
-  // ... `sha256:<signature>`? ... note that this could complicate key rotation
-  return signature;
+  const signature = base64url.decode(await hmac.sign({data}));
+  // multibase encode hash for future proofing
+  // 0x12 means sha2-256
+  // 32 is the digest length in bytes
+  return Buffer.concat([Buffer.from([0x12, 32]), signature]);
```

Comment:
Could we stick to one numeric base?  would `[18, 32]` work?

Reply:
We could, but I think it was more understandable this way since the multibase docs say 0x12 for the tag and then `32` is more understandable as decimal for the length. I will change it to `18` and add docs.